src/DiagramGenerator/Diagram/Board.php

```diff
@@ -127,8 +130,11 @@ public function drawBoard()
 
         // Add board texture
         if ($this->getBoardTexture()) {
-            $background = new \Imagick($this->getBackgroundTexture());
-            $textureSize = $this->getCellSize() * 2;
+            try {
+                $background = $this->getBackgroundTexture();
+            } catch (\ImagickException $exception) {
+                throw new CachedFileInvalidException();
```

Comment:
We can `throw new CachedFileInvalidException($exception->getMessage())`, and not catch it in controller. That way we will get rollbar info.